tasks/deploy/update.js

```diff
@@ -15,8 +16,11 @@ module.exports = function (gruntOrShipit) {
 
   function task() {
     var shipit = getShipit(gruntOrShipit);
+    _.assign(shipit.constructor.prototype, require('../../lib/releases'));
```

Comment:
You don't have to put that on the prototype, nobody will create a new Shipit object I think.